dashboard/history/tests/test_controller.py

```diff
@@ -1,23 +1,23 @@
 # -*- coding: utf-8 -*-
-import re
-from unittest.mock import patch
-
 import requests
 
+from re import search
```

Comment:
For ``re`` module, will be better to ``import re`` and use ``re.search`` in code - this is more readable, ``search`` does not tell that it use regex.

Reply:
changes to from re import search as re_search - should be clear to everyone